lark/load_grammar.py

```diff
@@ -1045,10 +1060,10 @@ def load_grammar(self, grammar_text, grammar_name='<?>', import_paths=[]):
 
         resolve_term_references(term_defs)
 
-        rules = rule_defs
+        ## Handle rules
```

Comment:
A very vague comment

Reply:
It just declares what that section does, similar to a function name